tendrl/alerting/api/manager.py

```diff
@@ -17,7 +16,7 @@
 app = Flask(__name__)
 plugin_manager = None
 persister = None
-
+HTTP_500_INTERNAL_SERVER_ERROR = 500
```

Comment:
@anmolbabu is this return code fine?? In Performance monitoring component the value is set as `1` I remember. Which one is correct?